cluster/manager/manager.go

```diff
@@ -1824,6 +1836,7 @@ func (c *ClusterManager) Remove(nodes []api.Node, forceRemove bool) error {
 			return errors.New(msg)
 		}
 
+	labForceRemoveNode:
```

Comment:
Why "lab"? That part of the name not immediately obvious. Include explanatory comment

Reply:
The "lab" stands for "label" -- there is a "goto" line above, that jumps to this label to skip a bunch of code.

Reply:
Renamed goto-label from ~labForceRemoveNode~ to `labelForceRemoveNode` in follow-up bfa12c9